app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -1,11 +1,42 @@
-import React from "react"
+import React from "react";
+import { Card, CardTitle, CardBody, CardText, CardLink, ListGroup, ListGroupItem } from "reactstrap";
+import { NavLink } from "react-router-dom"
+import apartments from '../mockApartments'
 
-const ApartmentIndex = () => {
+
+
+const ApartmentIndex = ({}) => {
   return (
+  
     <>
       <h3>ApartmentIndex</h3>
+        {apartments.map((apartment, index) => {
```

Comment:
Make this map statement conditional on there being apartments.

Reply:
added the conditional ? to the map statement.